src/unbound_script.rs

```diff
@@ -42,4 +52,28 @@ impl UnboundScript {
     }
     code_cache
   }
+
+  #[inline(always)]
+  pub fn get_source_mapping_url<'s>(
+    &self,
+    scope: &mut HandleScope<'s>,
+  ) -> Local<'s, Value> {
+    unsafe {
+      scope
+        .cast_local(|_| v8__UnboundScript__GetSourceMappingURL(self))
+        .unwrap()
```

Comment:
what if the script does not contain a source mapping url?

Reply:
It returns the `v8::Value` that is `undefined`. Just pushed a test.